pandas/_libs/tslibs/period.pyx

```diff
@@ -73,6 +73,7 @@ from pandas._libs.tslibs.offsets cimport (
 )
 from pandas._libs.tslibs.tzconversion cimport tz_convert_utc_to_tzlocal
 
+from .dtypes cimport PeriodDtype
```

Comment:
can you avoid all relative imports, its confusing.

Reply:
sure

Reply:
updated+green